### ISSUE TITLE
my bot doesn't walk anymore on pokebuddy.

### ISSUE THREAD

**Issue by jorgepsimoes** [finndev/PokeBuddy#313]
guys, help me please... 
after the niatic updated the pokemon go, i do logging with my account on pokebuddy but he don't walked anymore.
how to resolve that? 
if i play by nox or phone the game is working as well.
tks a lot my friends!

**Comment by ThiefthBooom** [comment]
The same happens to me

**Comment by rafael9690** [comment]
hack off

**Comment by TeemoOnShrooms** [comment]
Are you guys able to read or only to post random duplicate posts?

BOT aren't working anymore since last Niantics Update; people are working to fix that, no ETA no clue on when (or if) bots are coming to "life" again.
Keep yourselves updated, reading patiently, or at least stop spamming all the same things.

**Comment by Juan44** [comment]
Bots are down guys just wait

**Comment by TeemoOnShrooms** [comment]
https://www.reddit.com/live/xdkgkncepvcq

**Comment by jorgepsimoes** [comment]
tks my friends.. sorry by duplicate issue.

**Comment by Nexus-WoW** [comment]
yeah u cant see the 30 Issues before making a own issue for that.
